doc/rst/technotes/extern.rst

```diff
@@ -165,6 +167,15 @@ pointed-to type T should be one that is supported in C interoperability if the
 c_ptr(T) is used for C interoperability. The c_ptr(T) type supports
 indexing to get a reference to the i'th element (starting from 0).
 
+c_ptrConst(T)
+~~~~~~~~~~~~~
+
+The c_ptrConst(T) type is equivalent to c_ptr(T), except for disallowing
+mutation of the pointee. Like C, this does not change anything about the
```

Comment:
Consider "it disallows changing the pointed value"

Reply:
I like this, but with "pointed-to value" if that works as well